src/models/Review.java

```diff
@@ -6,15 +6,23 @@
 	private String userName;
 	private int upvotes;
 	private int downvotes;
-	private Location location;
+	private String locationName;
+	private String image;
 	
-	public Review(String title, String body, String userName, int upvotes, int downvotes, Location location) {
+	public Review(String title, 
+			String body, 
+			String userName, 
+			int upvotes, 
+			int downvotes, 
+			String locationName, 
```

Comment:
I changed the Review class to only have a location name, not an entire location object. It seems to me that if we need the entire location object, we can just make a request to get the location data.